### ISSUE TITLE
installation failed, Vagrant insecure key detected

### ISSUE THREAD

**Issue by tomkuipers** [OpenConext/OpenConext-deploy#109]
Running the installation script 'provision-vagrant' fails, complaining about Vagrant insecure key detected. See logs:

```
OpenConext-deploy-217.1 tomk$ ./provision-vagrant 
Bringing machine 'lb_centos7' up with 'virtualbox' provider...
Bringing machine 'apps_centos7' up with 'virtualbox' provider...
==> lb_centos7: Box 'CentOS-7.0' could not be found. Attempting to find and install...
    lb_centos7: Box Provider: virtualbox
    lb_centos7: Box Version: >= 0
==> lb_centos7: Box file was not detected as metadata. Adding it directly...
==> lb_centos7: Adding box 'CentOS-7.0' (v0) for provider: virtualbox
    lb_centos7: Downloading: https://build.surfconext.nl/vagrant_boxes/virtualbox-centos7.box
==> lb_centos7: Successfully added box 'CentOS-7.0' (v0) for 'virtualbox'!
==> lb_centos7: Importing base box 'CentOS-7.0'...
==> lb_centos7: Matching MAC address for NAT networking...
==> lb_centos7: Setting the name of the VM: OpenConext Engineblock Loadbalancer
==> lb_centos7: Clearing any previously set network interfaces...
==> lb_centos7: Preparing network interfaces based on configuration...
    lb_centos7: Adapter 1: nat
    lb_centos7: Adapter 2: hostonly
==> lb_centos7: Forwarding ports...
    lb_centos7: 22 (guest) => 2222 (host) (adapter 1)
==> lb_centos7: Running 'pre-boot' VM customizations...
==> lb_centos7: Booting VM...
==> lb_centos7: Waiting for machine to boot. This may take a few minutes...
    lb_centos7: SSH address: 127.0.0.1:2222
    lb_centos7: SSH username: vagrant
    lb_centos7: SSH auth method: private key
    lb_centos7: Warning: Remote connection disconnect. Retrying...
    lb_centos7: Warning: Remote connection disconnect. Retrying...
    lb_centos7: Warning: Remote connection disconnect. Retrying...
    lb_centos7: 
    lb_centos7: Vagrant insecure key detected. Vagrant will automatically replace
    lb_centos7: this with a newly generated keypair for better security.
    lb_centos7: 
    lb_centos7: Inserting generated public key within guest...
    lb_centos7: Removing insecure key from the guest if it's present...
    lb_centos7: Key inserted! Disconnecting and reconnecting using new SSH key...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
    lb_centos7: Warning: Authentication failure. Retrying...
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.
```

**Comment by quartje** [comment]
This is a bug in Vagrant 1.8.5. To fix it, either manually patch it according to the instructions here: https://github.com/mitchellh/vagrant/issues/7610, or upgrade the recently released Vagrant 1.8.6.

**Comment by tomkuipers** [comment]
Thanks, I've updated to 1.8.6 and the installation script continues. However not completely without errors.


```
PLAY [all] *********************************************************************

TASK [Read vars from secrets file] *********************************************
ok: [localhost]
ok: [192.168.66.99]
ok: [192.168.66.98]

PLAY [loadbalancer:php-apps:java-apps:storage] *********************************

TASK [setup] *******************************************************************
ok: [192.168.66.99]
fatal: [192.168.66.98]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh.", "unreachable": true}

TASK [common : set timezone] ***************************************************
fatal: [192.168.66.99]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh.", "unreachable": true}
	to retry, use: --limit @/Users/tomk/Documents/Vagrant/OpenConext/OpenConext-deploy-217.1/provision-vm.retry

PLAY RECAP *********************************************************************
192.168.66.98              : ok=1    changed=0    unreachable=1    failed=0   
192.168.66.99              : ok=2    changed=0    unreachable=1    failed=0   
localhost                  : ok=1    changed=0    unreachable=0    failed=0
```
Created an issue for this at https://github.com/OpenConext/OpenConext-deploy/issues/111
